### PR TITLE
solution

### PR DESCRIPTION
**Pull request by AnyoneClown**

### CODE REVIEW

File: app/test_main.py [hunk 1]
```diff
@@ -1 +1,27 @@
...
+        (True, False, "Not accessible"),
+        (False, True, "Not accessible"),
+        (False, False, "Not accessible")
+    ]
+)
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+def test_can_access_google_page(
+        mock_valid_google_url: bool,
+        mock_has_internet: bool,
```
Comment (pavlejviki) [inline app/test_main.py:19]: wrong type annotation for mock_valid_google_url and mock_has_internet

**Review by pavlejviki — CHANGES_REQUESTED** [review]

File: app/test_main.py [hunk 2]
```diff
@@ -1 +1,27 @@
...
+        (True, True, "Accessible"),
+        (True, False, "Not accessible"),
+        (False, True, "Not accessible"),
+        (False, False, "Not accessible")
+    ]
+)
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+def test_can_access_google_page(
+        mock_valid_google_url: str,
```
Comment (pavlejviki) [inline app/test_main.py:18]:
```suggestion
        mock_valid_google_url: mock.MagicMock,
```

File: app/test_main.py [hunk 3]
```diff
@@ -1 +1,27 @@
...
+        mock_has_internet: None,
```
Comment (pavlejviki) [inline app/test_main.py:19]:
```suggestion
        mock_has_internet: mock.MagicMock,
```

**Review by pavlejviki — CHANGES_REQUESTED** [review]

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!

**Pull request closed without merging**